diff --git a/node_modules/react-native-reanimated/android/src/main/java/com/swmansion/reanimated/NodesManager.java b/node_modules/react-native-reanimated/android/src/main/java/com/swmansion/reanimated/NodesManager.java
index 5db6885..ba2f7b8 100644
--- a/node_modules/react-native-reanimated/android/src/main/java/com/swmansion/reanimated/NodesManager.java
+++ b/node_modules/react-native-reanimated/android/src/main/java/com/swmansion/reanimated/NodesManager.java
@@ -344,6 +344,20 @@ public class NodesManager implements EventDispatcherListener {
   }
 
   public void updateProps(int viewTag, Map<String, Object> props) {
+    /*
+     * This is a temporary fix intended to address an issue where updates to properties
+     * are attempted on views that may not exist or have been removed. This scenario can
+     * occur in fast-changing UI environments where components are frequently added or
+     * removed, leading to potential inconsistencies or errors when attempting to update
+     * views based on outdated references
+     */
+    try {
+      View view = mUIManager.resolveView(viewTag);
+      if (view == null) return;
+    } catch (IllegalViewOperationException e) {
+      return;
+    }
+
     // TODO: update PropsNode to use this method instead of its own way of updating props
     boolean hasUIProps = false;
     boolean hasNativeProps = false;